Gemfile.lock

```diff
@@ -138,49 +138,49 @@ GEM
     method_source (1.0.0)
     mime-types (3.3.1)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2020.0512)
+    mime-types-data (3.2020.1104)
```

Comment:
### [mime-types-data](https://github.com/mime-types/mime-types-data/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.2020.0512 → 3.2020.1104

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **6** commits. See the full changes on [the compare page](https://github.com/mime-types/mime-types-data/compare/v3.2020.0512...v3.2020.1104).

These are the individual commits:
- (b554787) [Correction of my name in Contributing.md](https://github.com/mime-types/mime-types-data/commit/b55478709413d781ef38d8bedf30eea74cf6a6db)
- (96c9a9c) [Merge pull request #35 from YoranBrondsema/patch-1](https://github.com/mime-types/mime-types-data/commit/96c9a9cba597ae538b76cb87fa3a863d95a92855)
- (a0fd0c4) [Add application/x-compressed-zip](https://github.com/mime-types/mime-types-data/commit/a0fd0c49ebc34c584fe8f9ee615a3b7c764fdcf8)
- (0d026ec) [Update types/application.yaml](https://github.com/mime-types/mime-types-data/commit/0d026ecd7734f6b5a28dc1c862ed3b0152aac55a)
- (4533f61) [Merge pull request #36 from kaoru/add-x-compressed-zip](https://github.com/mime-types/mime-types-data/commit/4533f61b1e598de492b3699a60ec51cda452cb6d)
- (44bce29) [Update 3.2020.1104](https://github.com/mime-types/mime-types-data/commit/44bce294bac998cbd96958cc71b1ed172e5a41ee)


</details>